<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Connect_Disconect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Write">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviado</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Recebido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CheckSerial_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Read">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="Baud_Rate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Parâmetros da Comunicação</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_vel_R">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vel_R">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_texte">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-256</number>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_texte_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-256</number>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_vel_L">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vel_L">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Select_Robot">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Robô 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robô 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robô 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robô 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Robô 5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V_Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V_Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Girar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Girar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
